packages/tasks/src/local-apps.ts

```diff
@@ -77,6 +77,9 @@ function isMarlinModel(model: ModelData): boolean {
 function isTransformersModel(model: ModelData): boolean {
 	return model.tags.includes("transformers");
 }
+function isTgiModel(model: ModelData): boolean {
+	return model.tags.includes("text-generation-inference");
```

Comment:
Note: this will be much more reliable once this PR: https://github.com/huggingface-internal/moon-landing/pull/11356 (internal) is merged